File: mcp-server/packages/context-extractor-java/src/main/java/io/contextextractor/examples/Logger.java
package io.contextextractor.examples;

/**
 * Simple logging utility for testing.
 */
public class Logger {
    public void logInfo(String message) {
        System.out.println("INFO: " + message);
    }
    
    public void logError(String message) {
        System.out.println("ERROR: " + message);
    }
    
    public void logDebug(String message, int level) {
        System.out.println("DEBUG[" + level + "]: " + message);
    }
}

File: mcp-server/packages/context-extractor-csharp/tests/Examples/Logger.cs
namespace ContextExtractor.Tests.Examples;

/// <summary>
/// Simple logging utility for testing.
/// </summary>
public class Logger
{
    public void LogInfo(string message)
    {
        Console.WriteLine($"INFO: {message}");
    }

    public void LogError(string message)
    {
        Console.WriteLine($"ERROR: {message}");
    }

    public void LogDebug(string message, int level = 1)
    {
        Console.WriteLine($"DEBUG[{level}]: {message}");
    }
}
